main.go

```diff
@@ -131,6 +132,7 @@ func main() {
 	flag.StringVar(&logLevel, "log-level", logging.LogLevelDebug, `The verbosity of the logging. Valid values are "debug", "info", "warn", "error". Defaults to "debug".`)
 	flag.StringVar(&logFormat, "log-format", "text", `The log format. Valid options are "text" and "json". Defaults to "text"`)
 	flag.BoolVar(&autoScalingRunnerSetOnly, "auto-scaling-runner-set-only", false, "Make controller only reconcile AutoRunnerScaleSet object.")
+	flag.StringVar(&updateStrategy, "update-strategy", "immediate", "Immediately or eventually mutate resources on upgrade with running/pending jobs.")
```

Comment:
Might be nice to provide more hints in flag docs :relaxed: 
```suggestion
	flag.StringVar(&updateStrategy, "update-strategy", "immediate", `Strategy for mutating resources on upgrade with running/pending jobs. (values: "eventual", "immediately"; default: "immedately")`
```

Reply:
Fixed, thank you